cmd/model/transform.go

```diff
@@ -29,9 +29,10 @@ import (
 var transformCmd = &cobra.Command{
 	Use:   "transform",
 	Short: "Transforms an authorization model",
-	Example: `fga model transform --file=model.json --from-json
+	Example: `fga model transform --file=model.json --input-format json
```

Comment:
```suggestion
	Example: `fga model transform --file=model.json
```

Reply:
`--input-format` seems to still be requied

Reply:
I made it default to the format based on the extension and updated the help text